runtime/doc/options.txt

```diff
@@ -2650,6 +2650,12 @@ A jump table for the options with a short description can be found at |Q_op|.
 	To close folds you can re-apply 'foldlevel' with the |zx| command or
 	set the 'foldclose' option to "all".
 
+    						      *'foldoptions'* *'fop'*
+'foldoptions' 'fop'	string (default "")
+			local to window
+	List of words that change the behavor of the |foldcolumn|.
+          nodigits      Disable digits shown to indicate the nesting level.
```

Comment:
I would change this to `digits` and add it to the default.

Reply:
I've tried doing this but can't seem a find a way to properly set the default value. Doing it in the lua options definitions has no effect.

Maybe you have some pointers on this?

Reply:
> I've tried doing this but can't seem a find a way to properly set the default value. Doing it in the lua options definitions has no effect.

did you look at existing examples, e.g. https://github.com/neovim/neovim/blob/9370e1c5111ee90f64260398b0623da4759f8f16/src/nvim/options.lua#L633-L640

and what exactly did you try

Reply:
@justinmk 

Yeah, I've looked hard at finding a solution to my issue. But totally hit a wall.

Here's my uncomitted work based on this branch:

```patch
diff --git a/runtime/doc/options.txt b/runtime/doc/options.txt
index 536cce727..6a81e5dc2 100644
--- a/runtime/doc/options.txt
+++ b/runtime/doc/options.txt
@@ -2651,10 +2651,10 @@ A jump table for the options with a short description can be found at |Q_op|.
        set the 'foldclose' option to "all".

                                                      *'foldoptions'* *'fop'*
-'foldoptions' 'fop'    string (default "")
+'foldoptions' 'fop'    string (default "digits")
                        local to window
-       List of words that change the behavor of the |foldcolumn|.
-          nodigits      Disable digits shown to indicate the nesting level.
+       List of words that change the behavor of the |folding|.
+          digits        Enable digits shown to indicate the nesting level.

                                                *'foldtext'* *'fdt'*
 'foldtext' 'fdt'       string (default: "foldtext()")
diff --git a/src/nvim/option_defs.h b/src/nvim/option_defs.h
index a4de28f2c..36d1e9c0f 100644
--- a/src/nvim/option_defs.h
+++ b/src/nvim/option_defs.h
@@ -461,9 +461,9 @@ static char *(p_fdo_values[]) = { "all", "block", "hor", "mark", "percent",
 #define FDO_UNDO               0x200
 #define FDO_JUMP               0x400
 #ifdef IN_OPTION_C
-static char *(p_fop_values[]) = { "nodigits", NULL };
+static char *(p_fop_values[]) = { "digits", NULL };
 #endif
-#define FOP_NODIGITS           0x01
+#define FOP_DIGITS             0x01
 EXTERN char_u *p_fp;          // 'formatprg'
 EXTERN int p_fs;                // 'fsync'
 EXTERN int p_gd;                // 'gdefault'
diff --git a/src/nvim/options.lua b/src/nvim/options.lua
index 8bb4ad29d..afdb127bb 100644
--- a/src/nvim/options.lua
+++ b/src/nvim/options.lua
@@ -867,7 +867,7 @@ return {
       type='string', list='onecomma', scope={'window'},
       deny_duplicates=true,
       redraw={'current_window'},
-      defaults={if_true=''}
+      defaults={if_true='digits'}
     },
     {
       full_name='foldenable', abbreviation='fen',
diff --git a/src/nvim/screen.c b/src/nvim/screen.c
index 7562326f5..c73b195c6 100644
--- a/src/nvim/screen.c
+++ b/src/nvim/screen.c
@@ -1922,7 +1922,7 @@ static size_t fill_foldcolumn(char_u *p, win_T *wp, foldinfo_T foldinfo, linenr_
   int symbol = 0;
   int len = 0;
   bool closed = foldinfo.fi_lines > 0;
-  bool no_digits = wp->w_p_fop_flags & FOP_NODIGITS;
+  bool no_digits = !(wp->w_p_fop_flags & FOP_DIGITS);
   // Init to all spaces.
   memset(p, ' ', MAX_MCO * fdc + 1);

diff --git a/test/functional/ui/fold_spec.lua b/test/functional/ui/fold_spec.lua
index f73459038..f40999cfe 100644
--- a/test/functional/ui/fold_spec.lua
+++ b/test/functional/ui/fold_spec.lua
@@ -1866,7 +1866,7 @@ describe("folded lines", function()
         ]])
       end

-      command('set foldoptions=nodigits')
+      command('set foldoptions=')
       if multigrid then
         screen:expect([[
           ## grid 1
```

It doesn't seem like `defaults` does anything when the scope is set to `window`, unless I'm totally missing something obvious here :thinking:

Reply:
I studied the `cursorlineopt` option (and quite a few others), but it didn't really get me anywhere :(

> Maybe try redraw={'current_window_only'} (just a guess) ?

This has no effect.